js/colourpicker.js

```diff
@@ -153,16 +143,22 @@
       if( input.data('colourpicker-initialized') ) return;
 
       // Handle settings
-      settings = $.extend(true, {}, defaults, settings);
-
-	    // Palette type
-	    colourpicker.addClass('palette-' + settings.palette);
-	  
-	    // If adding a transparent checkbox, make the parent an input group
-      if( settings.allowTransparent ) {
-        colourpicker.addClass('input-group');
+      settings = $.extend(false, {}, defaults, settings);
+      //for backwards compatibility
+      if ($.type(settings.allowedCols) === "string") {
```

Comment:
I prefer to keep it. You never know, right? I'm already thinking I might use this in my own javascript projects, so maybe somebody out there secretly did the same thing, unlikely though that is since they'd probably be using mini-color. Still, it's a pretty minor thing.